libs/application-generic/src/usecases/create-workflow/create-workflow.usecase.ts

```diff
@@ -289,7 +305,7 @@ export class CreateWorkflow {
     templateSteps: INotificationTemplateStep[],
     trigger: INotificationTrigger,
     triggerIdentifier: string,
-  ) {
+  ): Promise<GetWorkflowByIdsResponseDto> {
```

Comment:
Could possibly rename `GetWorkflowByIdsResponseDto` to something more generic, but I think that's for a follow-up PR.

Reply:
Done.